core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,38 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var suppressAnswer = responsesService.getAnswerGroups();
+        var defaultAnswers;
+        // For choice length, have indices 0 through choice length - 1
+        for (var x = 0; x < choices; x++) {
+          if (suppressAnswer['' + x + ''] === undefined) {
+            // Return false if no object exists for the index
+            /* Note that multiple choice only allows for
```

Comment:
I don't think this is true? When you first create the answer group, you can only add one rule, but later on you can add additional rules to the same group (so that they result in the learner getting the same feedback),